components/Typography/typography.css

```diff
@@ -0,0 +1,101 @@
+.upper-alpha {
+    list-style-type: upper-alpha;
+    margin-left: 2rem;
+}
+
+.lower-alpha {
+    list-style-type: lower-alpha
+}
+
+.upper-roman {
+    list-style-type: upper-roman
+}
+
+.lower-roman {
+    list-style-type: lower-roman;
+}
+
+ul.inline-list {
+    display: flex;
+    justify-content: space-evenly;
+    margin-bottom: 1rem;
+}
+
+.headings-container {
+    width: 24rem;
+    height: 23rem;
+    border: 2px solid black;
+    margin-bottom: 1rem;
+}
+
+.Headings {
+    text-align: center;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.Headings h1 {
+    font-size: 3rem;
+}
+
+.Headings h2 {
+    font-size: 2.5rem;
+}
+
+.Headings h3 {
+    font-size: 2rem;
+}
+
+.Headings h4 {
+    font-size: 1.6rem;
+}
+
+.Headings h5 {
+    font-size: 1.2rem;
+}
+
+.text-container {
+    text-align: center;
+    height: 19rem;
+    width: 30rem;
+    border: 2px solid #323232;
+}
+
+.Texts-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.small-text {
+    font-size: 2rem;
+    margin: 0.5rem;
+}
+
+.medium-text {
+    font-size: 2rem;
+}
+
+.large-text {
+    font-size: 2.5rem;
+}
+
+.strike-Text {
```

Comment:
Naming of the class and the class seems not to be doing what is says.

Reply:
okkk noted